herddb-core/src/main/java/herddb/cluster/BookKeeperDataStorageManager.java

```diff
@@ -1259,8 +1309,9 @@ private static LogSequenceNumber readLogSequenceNumberFromIndexMetadataFile(Stri
     @Override
     public List<Index> loadIndexes(LogSequenceNumber sequenceNumber, String tableSpace) throws DataStorageManagerException {
         try {
-            String tableSpaceDirectory = getTableSpaceZNode(tableSpace);
-            ensureZNodeDirectory(tableSpaceDirectory);
+//          TODO
```

Comment:
Impl leftovers?

Reply:
actually this is no more needed.
I am dropping this commented code, thanks